src/pages/AddJob.jsx

```diff
@@ -42,6 +43,73 @@ export default function AddJob() {
     }
   );
 
+  useEffect(() => {
+    let hasdraft = window.localStorage.getItem("draft");
+    if (hasdraft !== null && toastid.length === 0) {
+      toastid.push(1);
```

Comment:
what do `toastid` suppose to do?

Reply:
Because this is a simple way to limit the number of visible toast to 1.